checker/manual/subtyping-checker.tex

```diff
@@ -181,11 +173,23 @@ \section{Subtyping Checker example\label{subtyping-example}\label{encrypted-exam
                      ^
 \end{Verbatim}
 
+\item
+You can also provide a set of directories that contain the qualifiers using the
+\code{-AqualDirs} option, and also adding the folders to the bootclass path, for example:
+
+\begin{Verbatim}
+  javac -processorpath myqualpath -Xbootclasspath/p:/full/path/to/myproject:/full/path/to/otherLib \
+        -processor org.checkerframework.common.subtyping.SubtypingChecker \
+        -AqualDirs=/full/path/to/myproject:/full/path/to/otherLib YourProgram.java
+\end{Verbatim}
+
+Note that in this example, myqual.Encrypted and myqual.PossiblyUnencrypted would exist in either the myproject folder or the otherLib folder.
```

Comment:
Same comment about tt italic font as above.  You can adjust the font within the Verbatim environment too.